mealie/routes/households/controller_group_recipe_actions.py

```diff
@@ -68,7 +68,9 @@ def delete_one(self, item_id: UUID4):
     # Actions
 
     @router.post("/{item_id}/trigger/{recipe_slug}", status_code=202)
-    def trigger_action(self, item_id: UUID4, recipe_slug: str, bg_tasks: BackgroundTasks) -> None:
+    def trigger_action(
+        self, item_id: UUID4, recipe_slug: str, bg_tasks: BackgroundTasks, scaled_amount: float = Body(..., embed=True)
```

Comment:
Should we default this to 1 to maintain old behavior?

```suggestion
        self, item_id: UUID4, recipe_slug: str, bg_tasks: BackgroundTasks, scaled_amount: float = Body(1, embed=True)
```

Reply:
Good idea, changed it